src/logging.cpp

```diff
@@ -17,7 +17,7 @@
 #include <set>
 
 // Unavoidable because these are in util.h.
```

Comment:
I know this is an old comment, but I'm not sure what its supposed to mean. What is unavoidable here? This comment is also in logging.h

Reply:
Oh wait now I see what it's referring to 
```suggestion
// externs unavoidable because these are in util.h.
```